Expensify/expensify-common
# Change default "Other" accounting icon

@tgolen will you please review? This PR is needed for the Web-Expensify PR here: https://github.com/Expensify/Web-Expensify/pull/28683

![image](https://user-images.githubusercontent.com/2319350/88312026-e07f7480-ccce-11ea-8c57-cee2e84677a2.png)


### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/129240

# Tests
1. What unit/integration tests cover your change? What autoQA tests cover your change?
1. What tests did you perform that validates your changed worked?

# QA
1. Go through the Collect/Control flow in the Inbox
1. Click on "Yes" for the "Sync with Accounting Card"
1. On the subsequent card, make sure the "Other" choice has a new, blue icon
